### PR TITLE
Add Counterspell Santiago

### PR DESCRIPTION
**Pull request by Monoverde888**
awesome event im really excited

### CODE REVIEW

**Comment by Monoverde888** [comment]
oops i should remove the package lock

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/santiago**](https://counterspell-qzgxgf7ky.hackclub.dev/santiago) |
|:-:|
|<a href="https://counterspell-qzgxgf7ky.hackclub.dev/santiago">Screenshot of /santiago</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/cX4jmJT2oo8CruIe0uOqEDNmXzSNGMEYBx7dHpCiVhSL8NwWdpGnNQ_ioMmpM5Be0lRuoFFJ8m1AE7Ic_hLuXQ.png">(view full size)</a> |







Commit 1a045a5eb661976d077a838ea009e89ddae1a82a (https://counterspell-qzgxgf7ky.hackclub.dev).

File: yarn.lock
Comment (samdev-7) [inline yarn.lock:1]: Could you revert these changes?

**Review by samdev-7 — CHANGES_REQUESTED** [review]
Almost there, some more changes and then it's good to go!

Also, are the flags supposed to be where they are? It looks a bit off when I scroll.

File: yarn.lock
Comment (Monoverde888) [reply yarn.lock:1]: done!

**Comment by Monoverde888** [comment]
and yeah the flags are kinda wonky but it doesnt really matter

**Review by samdev-7 — APPROVED** [review]

**Pull request merged**